.github/workflows/lint.yml

```diff
@@ -129,6 +129,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: DoozyX/clang-format-lint-action@v0.16.2
         with:
+          exclude: './sw/saris'
```

Comment:
Is this strictly necessary?

Reply:
Not strictly, no. But currently, the code is formatted for maximum readability of the iterations and point loop. The `clang-format` reformatting really tears it apart and makes it very hard to parse what's going on.

I do agree that this workflow level exclude is ugly; if you know of a way to do it with a pragma or something, we can do that.